Makefile

```diff
@@ -28,8 +28,14 @@ default_target: allsrc
 	bin/unicon -features >> unicon-features.log
 	@echo ======================================== >> unicon-features.log
 	@cat unicon-features.log
+ifeq ($(MSYSTEM),UCRT64)
+	@echo "To finish, do either \"make WinPatch\" or \"make install\""
+	@echo "   make WinPatch completes the installation in situ."
```

Comment:
Why is `make WinPatch` needed?  Binaries seems to already have the correct path already, here is what I got for example without having to patch:

```
Binaries at C:\msys64\home\WDAGUtilityAccount\unicon\bin\
```